.github/workflows/release-binaries.yaml

```diff
@@ -0,0 +1,33 @@
+name: Monstache release binaries
+
+on:
+  release:
+    types: [created]
```

Comment:
One example I saw for this initiated the action on a new tag.  How would that compare to how it's configured now?
https://github.com/goreleaser/goreleaser-action#run-on-new-tag

Reply:
Yes it updates the release and adds artifacts to it. 

You have to create your release like you seems to do usually selecting the tag and that should be it. The initial don't have to be a draft release. I have added you to my fork of monstache so that you can play with it if you want.

Reply:
@rwynn It can be done on new tag also no problem. There need to be some extra work to be done to make sure that the tag is in fact a release tag and not some other tag that's why I usually prefer to use this hook but let me know what you prefer. Anyway it can be changed at any moment.

Reply:
thanks for the explanation, let's go with on release.